test/player.js

```diff
@@ -145,7 +145,7 @@ describe("Player", () => {
     let event;
 
     beforeEach(() => {
-      event = { key: "", preventDefault: sinon.spy() };
+      event = { key: "", preventDefault: sinon.spy(), target: document.body };
```

Comment:
test for when target is not `document.body`?

Reply:
@emcniece all tests in this section are for when is `document.body` except for the last one that makes sure every remaining keyboard command gets through (even if it is `document.body`).